packages/framework-core/src/booster-token-verifier.ts

```diff
@@ -10,25 +10,22 @@ export class BoosterTokenVerifier {
 
   public constructor(private config: BoosterConfig) {
     if (this.config.tokenVerifier) {
-      const { issuer, jwksUri } = this.config.tokenVerifier
-      this.client = jwksRSA({
-        jwksUri,
-        cache: true,
-        cacheMaxAge: 900000, // 15 Minutes, at least to be equal to AWS max lambda limit runtime
-      })
+      if (this.config.tokenVerifier.jwksUri) {
+        this.client = jwksRSA({
+          jwksUri: this.config.tokenVerifier.jwksUri,
+          cache: true,
+          cacheMaxAge: 900000, // 15 Minutes, at least to be equal to AWS max lambda limit runtime
```

Comment:
Nitpicking here, why not:

```suggestion
          cacheMaxAge: 15 * 60 * 1000, // 15 Minutes, at least to be equal to AWS max lambda limit runtime
```